### ISSUE TITLE
SAGA fails to get job status from Bridges

### ISSUE THREAD

**Issue by iparask** [radical-cybertools/radical.saga#745]
While trying to run an RP example on Bridges, it failed with:
```
2019-10-16 06:41:46,293: pmgr.0000.launching.0: pmgr.0000.launching.0           : MainThread     : ERROR   : bulk launch failed
Traceback (most recent call last):
  File "/home/iparask/test_rp/lib/python3.5/site-packages/radical/saga/adaptors/slurm/slurm_job.py", line 1057, in _job_get_state
    match       = self.js.scontrol_jobstate_re.search(out)
AttributeError: 'SLURMJobService' object has no attribute 'scontrol_jobstate_re'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/iparask/test_rp/lib/python3.5/site-packages/radical/pilot/pmgr/launching/default.py", line 486, in work
    self._start_pilot_bulk(resource, schema, pilots)
  File "/home/iparask/test_rp/lib/python3.5/site-packages/radical/pilot/pmgr/launching/default.py", line 725, in _start_pilot_bulk
    if j.state == rs.FAILED:
  File "/home/iparask/test_rp/lib/python3.5/site-packages/radical/saga/job/job.py", line 625, in get_state
    return self._adaptor.get_state (ttype=ttype)
  File "/home/iparask/test_rp/lib/python3.5/site-packages/radical/saga/adaptors/cpi/decorators.py", line 62, in wrap_function
    return sync_function (self, *args, **kwargs)
  File "/home/iparask/test_rp/lib/python3.5/site-packages/radical/saga/adaptors/slurm/slurm_job.py", line 1117, in get_state
    self._state = self._job_get_state(self._id)
  File "/home/iparask/test_rp/lib/python3.5/site-packages/radical/saga/adaptors/slurm/slurm_job.py", line 1075, in _job_get_state
    "job %s:\n%s" % (pid,ex))
radical.saga.exceptions.NoSuccess: Error getting the job state for job 6516401:
'SLURMJobService' object has no attribute 'scontrol_jobstate_re' (/home/iparask/test_rp/lib/python3.5/site-packages/radical/saga/adaptors/slurm/slurm_job.py +1075 (_job_get_state)  :  "job %s:\n%s" % (pid,ex)))
pmgr.0000.launching.0.log (END)
```

The job was submitted correctly:
```
[paraskev@login005 ~]$ showq
             JOBID PARTITION     NAME     USER ST       TIME  NODES NODELIST(REASON)
           6516401  RM-small pilot.00 paraskev PD       0:00      1 (None)
[paraskev@login005 ~]$
[paraskev@login005 ~]$ showq
             JOBID PARTITION     NAME     USER ST       TIME  NODES NODELIST(REASON)
           6516401  RM-small pilot.00 paraskev  R       0:12      1 r003
[paraskev@login005 ~]$ scancel 6516401
```

**Comment by mturilli** [comment]
Confirmed also when using slurm on Stampede2

**Comment by iparask** [comment]
Fixed with `fix/issue_745` branch

**Comment by AymenFJA** [comment]
confirmed for me as well, on Comet. Working with @iparask on it now.

**Comment by iparask** [comment]
This is fixed through PR #748 . Closing the ticket.
